src/components/AmountInput.tsx

```diff
@@ -11,7 +11,7 @@ type Props = {
   label?: string;
   error?: string;
   brand?: Brand | null;
-  onChange: (value: bigint) => void;
+  onChange?: (value: bigint) => void;
```

Comment:
consider giving it a default noop implementation instead, so you don't have to condition the call

Reply:
Done